plugins/linear-genome-view/src/LinearGenomeView/components/LinearGenomeView.tsx

```diff
@@ -27,6 +28,27 @@ const useStyles = makeStyles()(theme => ({
 const LinearGenomeView = observer(({ model }: { model: LGV }) => {
   const { tracks, error, initialized, hasDisplayedRegions } = model
   const { classes } = useStyles()
+  const theme = useTheme()
+  const ref = useWidthSetter(model, theme.spacing(1))
```

Comment:
possibly this useWidthSetter here?

Reply:
might be able to convert it to a plain useRef?

Reply:
yep, just found both of those. that was it